packages/go/analysis/analysis.go

```diff
@@ -240,10 +239,10 @@ func GetLAPSSyncers(tx graph.Transaction, domain *graph.Node) ([]*graph.Node, er
 	}
 }
 
-func GetDCSyncers(tx graph.Transaction, domain *graph.Node, filterTierZero bool) ([]*graph.Node, error) {
+func GetDCSyncers(tx graph.Transaction, domain *graph.Node) ([]*graph.Node, error) {
 	var (
-		getChangesQuery    = fromEntityToEntityWithRelationshipKind(tx, domain, ad.GetChanges, filterTierZero)
```

Comment:
It looks like the `fromEntityToEntityWithRelationshipKind` also has no need for filterTierZero as every use case after your change issues `false`. What do you think of cleaning that up as well?

Reply:
Good catch! I removed that parameter and logic in the latest push.